pkgs/applications/virtualization/colima/default.nix

```diff
@@ -14,13 +14,25 @@ buildGoModule rec {
     owner = "abiosoft";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-g7q2DmtyArtW7Ii2XF5umXQ0+BlCSa1Q7VNNuIuX65k=";
+    sha256 = "sha256-KYW3gxf21aWnuRHkysOjArzMSNH3m3XDoi6Sic3N+Po=";
+
+    # We need the git revision
+    leaveDotGit = true;
```

Comment:
Think this is still dangerous to use and the hash will change randomly :(

See #8567

Reply:
Look closer, I delete the .git dir in the next few lines

Reply:
Oops! Looksgood

Reply:
Hah I feel silly right now. Obviously skimmed this, didn't pay attention, and then ended up proposing a similar change in #8567 which is already what you're doing here. >.>